src/main/java/blackjack/view/OutputView.java

```diff
@@ -49,7 +49,7 @@ public static void printPlayerHandsMessage(PlayerDto dto) {
 
     private static String printCards(List<Card> cards) {
         List<String> cardSignatures = cards.stream()
-                .map(card -> card.getSignature() + card.getSuit().getName())
+                .map(card -> card.getSymbol() + card.getSuit().getName())
```

Comment:
```suggestion
                .map(card -> card.getSymbol() + card.getSuitName())
```
요렇게 해주면 조금 더 깔끔하겠네요 :)

Reply:
넵 그러네요!